components/ContractDataTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { /* useMantineTheme, */ createStyles, Text, Table, Badge } from "@mantine/core";
+
+type ContractData = {
+    address: string;
+    contractDeployer: string;
+    deployedBlockNumber: number;
+    name: string;
+    symbol: string;
+    totalBalance: number;
+};
+
+type ContractDataTableProps = {
+    contractAddresses: ContractData[];
+};
+
+const useStyles = createStyles((theme) => ({
+    table: {
+        padding: "3rem",
+        textAlign: "center"
+    },
+
+    tableHeading: {
+        textAlign: "center"
+    },
+}));
+
+export default function ContractDataTable({ contractAddresses }: ContractDataTableProps) {
+    // const theme = useMantineTheme();
+    const { classes } = useStyles();
+
+    return (
+        <div className={classes.table}>
```

Comment:
Place inside a Mantine container. It will give proper margins

Reply:
https://mantine.dev/core/container/

Reply:
This one is completed.